lib/binary/binary.nit

```diff
@@ -148,11 +148,15 @@ redef abstract class Reader
 	super BinaryStream
 
 	# Read a single byte and return `true` if its value is different than 0
+	#
+	# Returns `false` when an error is pending (`last_error != null`).
 	fun read_bool: Bool do return read_byte != 0
 
 	# Get an `Array` of 8 `Bool` by reading a single byte
 	#
 	# To be used with `BinaryWriter::write_bits`.
+	#
+	# Returns an array of `false` when an error is pending (`last_error != null`).
```

Comment:
By the contract stated in the synopsis, the array is of 8 `Bools`. So to avoid problems on treating the result as such, it is filled with `false`.